apps/tsilhqotin-language-hub/src/Configs/global.config.ts

```diff
@@ -20,13 +20,16 @@ export type SongIdToCredits = Record<string, string>;
 
 export type VideoIdtoCredits = Record<string, string>;
 
+export type TermOfTheDay = Record<string, string>;
+
 export type GlobalConfig = {
     siteTitle: string;
     funderInfos: FunderInfo[];
     // TODO Use this in the component
     linkInfos: LinkInfo[];
     songIdToCredits: SongIdToCredits;
     videoIdToCredits: VideoIdtoCredits;
+    termOfTheDay: TermOfTheDay;
```

Comment:
Let's call this `termOfTheDayConfig`. Same for the type. The current naming makes it sound like you've already searched and found the termID for today.

Reply:
Changed to termOfTheDayConfig.